#!/bin/sh /usr/share/dpatch/dpatch-run
## 339-magic-add-scribus.dpatch by Werner Fink <werner@suse.de>
##
## DP: Adds new magics for Scribus.

@DPATCH@

diff -Naurp file-4.24.orig/magic/Magdir/wordprocessors file-4.24/magic/Magdir/wordprocessors
--- file-4.24.orig/magic/Magdir/wordprocessors	2008-05-21 09:49:43.000000000 +0000
+++ file-4.24/magic/Magdir/wordprocessors	2008-05-21 09:50:54.000000000 +0000
@@ -153,3 +153,9 @@
 # From: Jamie Thompson <debian-bugs@jamie-thompson.co.uk>
 0	string/cB	\<map\ version	Freemind document
 !:mime	application/x-freemind
+
+# Type:        Scribus
+# From:        Werner Fink <werner@suse.de>
+0	string	\<SCRIBUSUTF8\ Version		Scribus Document
+0	string	\<SCRIBUSUTF8NEW\ Version	Scribus Document
+!:mime	application/x-scribus
